### ISSUE TITLE
Medaka step fails in the -profile fastq_test

### ISSUE THREAD

**Issue by hoelzer** [replikation/poreCov#240]
I'm trying to run the pipeline on a fresh setup on a Macbook Air M2 (not sure if the error is due to that). 

java -version
java version "17.0.4.1" 2022-08-18 LTS

nextflow -version
      N E X T F L O W
      version 22.10.0 build 5826
      created 13-10-2022 05:44 UTC (07:44 CEST)

## Command used
```bash
nextflow run replikation/poreCov -r 1.7.1 -profile local,docker,test_fastq -w work
```

## Error code

The pipeline just fails at the step:

`[b2/0e5d3a] process > artic_ncov_wf:artic_medaka (1)                       [100%] 1 of 1, failed: 1 ✔`

Checking the `.command.log` file reveals:

```
Command failed:medaka consensus --model r941_min_hac_g507 --threads 2 --chunk_len 800 --chunk_ovlp 400 --RG nCoV-2019_1 SARSCoV2.trimmed.rg.sorted.bam SARSCoV2.nCoV-2019_1.hdf
```

Full log file: 
[command.log](https://github.com/replikation/poreCov/files/9792065/command.log)

I also tried to run the command manually via:

```bash
docker run --rm -it -v $PWD:$PWD -w $PWD nanozoo/artic:1.3.0-dev--2c5b6a9 /bin/bash

medaka consensus --model r941_min_hac_g507 --threads 2 --chunk_len 800 --chunk_ovlp 400 --RG nCoV-2019_1 SARSCoV2.trimmed.rg.sorted.bam SARSCoV2.nCoV-2019_1.hdf
2022-10-15 12:34:22.705561: F tensorflow/core/lib/monitoring/sampler.cc:42] Check failed: bucket_limits_[i] > bucket_limits_[i - 1] (0 vs. 10)
qemu: uncaught target signal 6 (Aborted) - core dumped
Aborted
```

Not sure what's going on... and if it is bc/ of some Mac specifics.

Thanks!

**Comment by hoelzer** [comment]
I think this might be a mac-specific (M2 chip) issue w/ medaka... If the test_fastq profile is working for you guys please feel free to close that issue. If I can find a solution I will re-open/send a PR

**Comment by replikation** [comment]
@DataSpott ?

**Comment by replikation** [comment]
```
nextflow run replikation/poreCov -r 1.7.1 -profile local,docker,test_fastq -w work
N E X T F L O W  ~  version 22.04.3
Pulling replikation/poreCov ...
 downloaded from https://github.com/replikation/poreCov.git
Launching `https://github.com/replikation/poreCov` [big_bohr] DSL2 - revision: 82f9bc9171 [1.7.1]

________________________________________________________________________________
    
poreCov | A Nextflow SARS-CoV-2 workflow for nanopore data
    
 
  Latest available poreCov release: 1.7.1
  If neccessary update via: nextflow pull replikation/poreCov
________________________________________________________________________________
 Using 6/24 CPU threads per process for a local run [--max_cores]
 

.
Profile:             local,docker,test_fastq
Current User:        replikation
Nextflow-version:    22.04.3
poreCov-version:     1.7.1

Pathing:
Workdir location [-work-Dir]:
    /tmp/work
Output dir [--output]: 
    results
Databases location [--databases]:
    databases
Singularity cache dir [--cachedir]: 
    singularity_images
______________________________________
Parameters:
Medaka model:         r941_min_hac_g507 [--medaka_model]
Min depth nucleotide: 20 [--min_depth]
Latest Pangolin/Nextclade?: false [--update]
CPUs to use:          6 [--cores]
Memory in GB:         12 [--memory]
______________________________________


Primerscheme:        V1200 [--primerV]
Min read-length set to: 400 [--minLength]
Max read-length set to: 1500 [--maxLength]
______________________________________


RKI output for german DESH upload:
Output stored at:      results/4.RKI-summary  
Min Identity to NC_045512.2: 0.90 [--seq_threshold]
Min Depth used:        20 [--min_depth]
   Min Depth should be 20 or more for RKI upload
Proportion cutoff N:   0.05 [--n_threshold]
______________________________________

executor >  local (17)
[12/15e851] process > get_nanopore_fastq                       [100%] 1 of 1 ✔
[b5/2da460] process > read_qc_wf:nanoplot                      [100%] 1 of 1 ✔
[74/3805de] process > filter_fastq_by_length                   [100%] 1 of 1 ✔
[91/03befe] process > read_classification_wf:download_datab... [100%] 1 of 1 ✔
[6a/a52ddb] process > read_classification_wf:kraken2           [100%] 1 of 1 ✔
[d4/6baad5] process > read_classification_wf:krona             [100%] 1 of 1 ✔
[18/9519ee] process > artic_ncov_wf:artic_medaka (1)           [100%] 1 of 1 ✔
[bf/cb195c] process > artic_ncov_wf:covarplot (1)              [100%] 1 of 1 ✔
[1e/c4cc92] process > determine_lineage_wf:pangolin (1)        [100%] 1 of 1 ✔
[12/269e02] process > determine_mutations_wf:nextclade (1)     [100%] 1 of 1 ✔
[c2/4ac039] process > genome_quality_wf:president (1)          [100%] 1 of 1 ✔
[d7/abe05b] process > genome_quality_wf:seqrs (1)              [100%] 1 of 1 ✔
[fb/bd467d] process > rki_report_wf:rki_report (1)             [100%] 1 of 1 ✔
[4a/7b1ca4] process > align_to_reference (1)                   [100%] 1 of 1 ✔
[8b/87ac44] process > create_summary_report_wf:get_variants... [100%] 1 of 1 ✔
[56/85df99] process > create_summary_report_wf:plot_coverag... [100%] 1 of 1 ✔
[64/b252bf] process > create_summary_report_wf:summary_repo... [100%] 1 of 1 ✔
Completed at: 19-Oct-2022 13:15:43
Duration    : 12m 35s
CPU hours   : 1.7
Succeeded   : 17
```

@hoelzer seems to work, closing this.
